src/components/Card/Card.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  disabled?: boolean;
+  error?: boolean;
+  imgSrc: string;
+  imgAlt: string;
+  tagColor?: string;
+  tagBg?: string;
+};
+
+export const Card = ({
+  disabled = false,
+  error = false,
+  imgSrc,
+  imgAlt,
+  tagColor = 'blue800',
+  tagBg = 'blue100',
+}: Props) => {
+  tagColor = error ? 'red800' : tagColor;
+  tagBg = error ? 'red100' : tagBg;
```

Comment:
Dopuszczasz teraz, że `tagColor` i `tagBg` będą takie same przez co teks będzie nieczytelny. To co zrobiłeś z `error` to dobry trop. Dobrze gdyby `tag` przyjmował typ np.:
```
type tagType = 'success' | 'error' | 'warning'; // i tak dalej 
```
czyli to czym jest `State` w Figmie.
<img width="515" alt="Screen Shot 2021-09-29 at 11 59 28" src="https://user-images.githubusercontent.com/7902324/135247996-a0489eae-0259-4aa8-a23c-e4499a8ac7f1.png">

plus jakiś modyfikator, który ustawi jego wariant, np.:
```
type tagVariants =  'default' | 'minor' | 'light';
```
<img width="100" alt="Screen Shot 2021-09-29 at 11 59 37" src="https://user-images.githubusercontent.com/7902324/135248334-30d12126-22cb-49d3-97fc-badb713c0e27.png">

Dalej ustawiać modyfikator (BEM-owy) na `tag` i odpowiednie style, np. z wykorzystaniem `@each` w SCSS.

Reply:
Właśnie nad tym najbardziej się zastanawiałem, nie miałem pomysłu nad logicznym rozwiązaniem.